mmcv/runner/base_runner.py

```diff
@@ -414,12 +414,23 @@ def register_logger_hooks(self, log_config):
                 info, HOOKS, default_args=dict(interval=log_interval))
             self.register_hook(logger_hook, priority='VERY_LOW')
 
+    def register_itertimer_hook(self, itertimer_config):
+        if itertimer_config is None:
+            return
+        if isinstance(itertimer_config, dict):
```

Comment:
We can deepcopy the config first.

Reply:
Using mutable types as the default value is unsafe, since it can be modified in the method. When this is unavoidable, usually we deepcopy it before it is used.

Reply:
Thanks for your great comments!